# Upstream status: in 2.6.18-rc2

This patch fixes a typo in arch/mips/sgi-ip22/ip22-time.c, leading to
the incorrect year being set into the RTC chip.

Signed-off-by: Julien BLACHE <jb@jblache.org>

--- a/arch/mips/sgi-ip22/ip22-time.c	2006-07-08 22:17:02.000000000 +0000
+++ b/arch/mips/sgi-ip22/ip22-time.c	2006-07-08 22:17:29.000000000 +0000
@@ -76,7 +76,7 @@
 	save_control = hpc3c0->rtcregs[RTC_CMD] & 0xff;
 	hpc3c0->rtcregs[RTC_CMD] = save_control | RTC_TE;
 
-	hpc3c0->rtcregs[RTC_YEAR] = BIN2BCD(tm.tm_sec);
+	hpc3c0->rtcregs[RTC_YEAR] = BIN2BCD(tm.tm_year);
 	hpc3c0->rtcregs[RTC_MONTH] = BIN2BCD(tm.tm_mon);
 	hpc3c0->rtcregs[RTC_DATE] = BIN2BCD(tm.tm_mday);
 	hpc3c0->rtcregs[RTC_HOURS] = BIN2BCD(tm.tm_hour);
